### ISSUE TITLE
Publish a release

### ISSUE THREAD

**Issue by someone-somenet-org** [alliedmodders/amxmodx#1102]
1.8.2 is like 12 years old.

Please publish some 1.8.3 release to sum up the changes that happened in the last years.

**Comment by tenub** [comment]
gaben pls hl3

**Comment by rbradev** [comment]
Version [1.10](https://www.amxmodx.org/downloads-new.php?branch=master) is there and works for 25th anniversary

**Comment by rtxa** [comment]
Agreed with this. There's no good reason to delay this any further. If it's about compatibility, people can always fallback to older versions. If the current maintainers don't have the time to dedicate to the project, then we should look into bringing on new maintainers who can take care of it.
